catalyst/dl/experiment/base.py

```diff
@@ -13,9 +13,10 @@ class BaseExperiment(Experiment):
     Super-simple one-staged experiment
         you can use to declare experiment in code
     """
+
     def __init__(
         self,
-        model: Model,
+        model: Union[Model, Dict[str, Model]],
```

Comment:
no sure if we need such hack, why not to use nn.ModuleDict instead?

Reply:
I suggest to leave it as it is for now, for compatibillity.